test/integration/contractor_sees_correct_data_on_project_checkout_show_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.